.github/workflows/security.yml

```diff
@@ -17,6 +17,25 @@ defaults:
     shell: bash
 
 jobs:
+  set-up-trivy:
+    name: Set up Trivy
+    runs-on: ubuntu-22.04
+    steps:
+    - name: Setup Trivy vulnerability scanner
+      run: |
+        mkdir -p /home/runner/trivy
```

Comment:
this install process seems quite complicated, is there a reason we didnt use the deb?

https://aquasecurity.github.io/trivy/v0.55/getting-started/installation/

```
sudo apt-get install wget apt-transport-https gnupg
wget -qO - https://aquasecurity.github.io/trivy-repo/deb/public.key | gpg --dearmor | sudo tee /usr/share/keyrings/trivy.gpg > /dev/null
echo "deb [signed-by=/usr/share/keyrings/trivy.gpg] https://aquasecurity.github.io/trivy-repo/deb generic main" | sudo tee -a /etc/apt/sources.list.d/trivy.list
sudo apt-get update
sudo apt-get install trivy
```

Reply:
Not much, I just figured that since we are caching the vuln DB, why not caching trivy itself as well. Also it should be little faster, maybe this is why the k8s team used it like this.

Since I will be caching the vuln DB in the repo scan, and that uses the GitHub action so it doesn't download Trivy, I plan on using the deb in each `test-snap` run.

Reply:
I would suggest we use the deb for both runs for consistency, we are not speed sensitive on this run.